tests/app/celery/test_research_mode_tasks.py

```diff
@@ -21,15 +26,28 @@
 )
 
 
-def test_make_sns_callback(notify_api, rmock):
-    phone_number = "07700900001"
-    endpoint = "http://localhost:6011/notifications/sms/sns"
-    rmock.request("POST", endpoint, json="some data", status_code=200)
-    send_sms_response("sns", "1234", phone_number)
+@pytest.mark.parametrize(
+    "phone_number, sns_callback, sns_callback_args",
+    [
+        ("07700900000", sns_success_callback, {}),
+        ("07700900001", sns_success_callback, {}),
+        ("07700900002", sns_failed_callback, {"provider_response": "Phone is currently unreachable/unavailable"}),
+        ("07700900003", sns_failed_callback, {"provider_response": "Phone carrier is currently unreachable/unavailable"}),
+    ],
```

Comment:
Those numbers are not valid if I remember? Can you update tests with other numbers?

Reply:
I thought about it, these numbers are used everywhere in the tests and we could change these in a sweep. Per se, they are valid on the code that is tested, which is not the whole pipeline including validation, but they don't represent what we'd have in our deployed Canadian application.